react/OrderPayment.tsx

```diff
@@ -45,6 +45,8 @@ interface Context {
   setCardLastDigits: (cardDigits: string) => void
   paymentsValid: boolean
   setCardFormFilled: (filled: boolean) => void
+  isFreePurchase: boolean
+  lastStepsValid: boolean
```

Comment:
I think we can keep this `lastStepsValid` logic inside `checkout-payment`. I don't see it as a part of `order-payment` and only as a logic for rendering the free purchase alert or not

Reply:
I did it because this [component](https://github.com/vtex-apps/checkout-payment/blob/master/react/PaymentSummary.tsx#L26) already consumes the orderPayment. If I put it there I will need to consume from `useOrderForm`.  But I can do it if you think it's a better approach.

Reply:
yeah, I think this belongs there more than it does here